spec/requests/categories/update_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe 'Category update', type: :request do
+  let(:category) do
+    create(:category, name: 'Some category')
+  end
+
+  context 'with valid params' do
+    let(:params) do
+      { category: { name: "Another category" } }
+    end
+
+    it 'updates category ' do
+      patch "/categories/#{category.id}", params: params
+
+      expect(response).to redirect_to(categories_path)
+      follow_redirect!
+
+      category = Category.last
```

Comment:
In update specs you should use entity from let to check that is was changed. But let caches entities so you need to reload it to check that attributes changed. So your code should look like

```suggestion
      category.reload
```